### ISSUE TITLE
Bridge: if iPad goes to sleep process stops

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2726]
Not sure if you can even do anything about it, but if I'm downloading a database through Bridge and the iPad falls asleep (which happens with heavy databases, as has just happened to me with PACHA) the process stops and once I get back in after logging into the iPad I have to restart it. Annoying and disruptive, but again, not sure you would be able to change it or if it's an Apple thing.

**Comment by urapadmin** [comment]
no, at least to my knowledge I cannot keep the iPad from falling asleep. Why it stops the ongoing download when it falls to sleep is a mystery to me. It should not do that. 
However, can you set the "auto-lock" higher? It is under Display and Brightness. The highest is 15 Minutes, though. Even that cuts it short for the PACHA hulk.

**Comment by urapadmin** [comment]
if that helps, it might be worth mentioning in the Q&A.

**Comment by luizaogs** [comment]
Oh yeah I didn’t mean stopping the iPad from falling asleep, I meant stopping the process from stopping (oof) when it does.

**Comment by urapadmin** [comment]
that is a process that runs in one of Apple's blackboxes. So keeping the iPad from falling asleep it would be. I will look into solutions, nonetheless. Perhaps I can pretend to show a video.
